### PR TITLE
docs ADR

### LINKED ISSUE #2744: docs: write an ADR
- detail history of decisions made & why they're designed the way they are
- https://www.divio.com/blog/documentation/
- number 035 according to #2313

### PR DESCRIPTION
**Pull request by zramsay**
closes #2744 

* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: docs/architecture/adr-035-documentation.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Because of the organizational seperation between Tendermint & Cosmos, there is a challenge of "what goes where" for certain aspects of documentation.
+
+### Positive
+
+This architecture is largely positive relative to prior docs arrangements.
+
+### Negative
+
+A significant portion of the docs automation / build process is in private repos with limited access/visibility to devs. However, these tasks are handled by the SRE team.
+
```
[the remark this answers is not in the document]
Comment (zramsay) [reply docs/architecture/adr-035-documentation.md]: there's a commit in the docs that show the latest version, is that what you mean?

[the remark this answers is not in the document]
Comment (zramsay) [reply docs/architecture/adr-035-documentation.md]: https://tendermint.com/docs/#version

File: docs/architecture/adr-035-documentation.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+This is done while maintaining synchrony between the docs and code, i.e., the website is built whenever the docs are updated.
+
+## Status
+
+The two points above have been implemented; the `config.js` has a Google Analytics identifier and the documentation workflow has been up and running largely without problems for several months. Details about the documentation build & workflow can be found [here](../DOCS_README.md)
+
+### TODO
+
+- Organize & improve the Tendermint documentation using: https://www.divio.com/blog/documentation/
```
[the remark this answers is not in the document]
Comment (zramsay) [reply docs/architecture/adr-035-documentation.md]: sure, i agree

File: docs/architecture/adr-035-documentation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (melekes) [reply docs/architecture/adr-035-documentation.md]: screenshot 2018-11-15 at 20 05 00

[the remark this answers is not in the document]
Comment (melekes) [reply docs/architecture/adr-035-documentation.md]: I.e. I can switch to `stable` version

[the remark this answers is not in the document]
Comment (zramsay) [reply docs/architecture/adr-035-documentation.md]: oh i see what you mean. yeah, that was a nice feature of RTD, a bit harder to implement with the  current docs automation

**Comment by zramsay** [comment]
I've made the two negative you added issues in the devops repo, and the TODOs are all issues in the tendermint repo already

**Comment by ebuchman** [comment]
Oh - there was no mention of https://www.divio.com/blog/documentation/

Do we want to include that? We should probably spend some time thinking about how our existing documentation could be improved to better match what's outlined there

**Pull request merged**